composer.json

```diff
@@ -42,7 +43,6 @@
     "bin": [
         "bin/changelog-generator"
     ],
-    "platform": {
-        "php": "7.1.x"
-    }
+    "minimum-stability": "dev",
```

Comment:
Not really a fan here. Whenever dev version is required, `@dev` override should be used on per-dependency level, not globally.